prerace_setup.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from animal_factory import factory
+from config import RACERS_CONFIG_PATH
+from constants import *
```

Comment:
do you use every constant in all modules? If a constant will only be used in one module, you can move it to inside the module